test/consts/helm.go

```diff
@@ -1,4 +1,4 @@
 package consts
 
 // KongHelmChartVersion is the version of the Kong Helm chart to use in tests.
-const KongHelmChartVersion = "2.26.3"
+const KongHelmChartVersion = "2.29.0"
```

Comment:
This should be handled by https://github.com/Kong/kubernetes-ingress-controller/pull/4846 (which will put this into `.github/test_dependencies.yaml`)

Reply:
But I don't think it should be a blocker to merge this PR.